filter/specItemFilter.go

```diff
@@ -70,7 +70,7 @@ func sanitize(tag string) string {
 	if _, err := strconv.ParseBool(tag); err == nil {
 		return fmt.Sprintf("{%s}", tag)
 	}
-	return tag
+	return strings.ToLower(tag)
```

Comment:
Thanks for the PR @haroon-sheikh can we add a new property to make this a configuration? By default it's case sensitive with the option to turn it off. Worth observing if anyone uses case sensitive tag and making case insensitivity the default option in a future release.

Reply:
My only concern is, if we turn it off by default? then its very unlikely to be used by others and we'll never find out if they're using it in a case sensitive way.

Reply:
I've now added a flag around it. Let me know what you think?